### PR TITLE
fix: Unable to create a file in /data folder

### PR DESCRIPTION
**Pull request by iannbing**
## Description

This PR unifies the FE logic of creating a step, and thus fixed the following bugs:

- When creating a file in `/data` folder and check "Create a step", the file assigned to the step is incorrect.
- When dropping a file to create a step, the `Step Name` should be auto-focused.

## Checklist

- [x] I have manually tested my changes and I am happy with the result.
- [x] The PR branch is set up to merge into `dev` instead of `master`.

### CODE REVIEW

**Review by yannickperrenet — APPROVED** [review]
Tested and worked like a charm :man_dancing:

File: services/orchest-webserver/client/src/pipeline-view/file-manager/ActionBar.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ export function ActionBar({
       reload();
 
       if (file.shouldCreateStep) {
-        createStep(file.projectPath);
+        createStep(file.fullPath);
```
Comment (mausworks) [inline services/orchest-webserver/client/src/pipeline-view/file-manager/ActionBar.tsx:51]: Are you supposed to be able to have steps in the data folder?

**Review by mausworks — APPROVED** [review]
Just a question! Otherwise LGTM. ✌️

File: services/orchest-webserver/client/src/pipeline-view/file-manager/ActionBar.tsx (hunk 1 above)
Comment (iannbing) [reply services/orchest-webserver/client/src/pipeline-view/file-manager/ActionBar.tsx:51]: @mausworks  Yes.

Comment (ricklamers) [reply services/orchest-webserver/client/src/pipeline-view/file-manager/ActionBar.tsx:51]: If you mean can steps point to file in `/data` the answer is yes. In the pipeline definition they store the absolute path including the `/data` prefix.

**Pull request merged**